configuration/pipeline_config.json

```diff
@@ -3,6 +3,7 @@
   "RawDataSources": [
     {
       "SourceType": "RapidPro",
+      "SourceName": "Jambo",
```

Comment:
```suggestion
      "SourceName": "jambo",
```To match the stringvalue

Reply:
Jambo matches the match_value, which is what it's supposed to equal